cime_config/config_archive.xml

```diff
@@ -7,6 +7,9 @@
     <!-- match filenames of the form
          h.static[._optional instance number].nc[.optional tile number] -->
     <hist_file_extension>h\.static(\._\d*)?\.nc(\.\d*)?$</hist_file_extension>
+    <!-- match filenames of the form
+         h.bgc.*[._optional instance number].nc -->
+    <hist_file_extension>h\.bgc\..*?.?[_\d+]+.nc$</hist_file_extension>
```

Comment:
It turns out this block **is** necessary -- without it, `h.bgc.native`, `h.bgc.daily`, and other `h.bgc.*` streams are not archived. I wonder if one fix would be to change lines 5 and 6:

```diff
    <!-- match filenames of the form
-         h[.optional stream name][._optional instance number].(date string).nc[.optional tile number] -->
-    <hist_file_extension>h(\.\w+)?(\._\d*)?\.[-\d+]+\.nc(\.\d*)?$</hist_file_extension>
+         h.stream_name.[.optional stream name][._optional instance number].(date string).nc[.optional tile number] -->
+    <hist_file_extension>h\.\w+(\.\w+)?(\._\d*)?\.[-\d+]+\.nc(\.\d*)?$</hist_file_extension>
```

We don't generate a `mom6.h`, do we? I'm going to leave this file as-is, but we can revisit it after it's merged -- I don't want to get side-tracked going down a rabbit-hole of testing various possible changes when this works and removing it doesn't.